mobilecoind-dev-faucet/src/lib.rs

```diff
@@ -247,6 +257,9 @@ impl State {
         &self,
         req: &JsonFaucetRequest,
     ) -> Result<api::SubmitTxResponse, String> {
+        if !self.worker.activate() {
```

Comment:
Would it make sense to centralize this logic somewhere? Maybe I'm thinking about this in the wrong way, but it feels like if we want it to be the case that if we activate the worker on any type of post request, we should be doing that in the rocket code to call activate on post requests instead of having it activate on these specific functions.

Reply:
we could make a "self.activate" function, sure